firebase-firestore/src/main/java/com/google/firebase/firestore/MemoryLruGcSettings.java

```diff
@@ -30,20 +37,42 @@ public MemoryLruGcSettings build() {
       return new MemoryLruGcSettings(sizeBytes);
     }
 
-    public void setSizeBytes(long size) {
+    /**
+     * Sets an approximate cache size threshold for the memory cache. If the cache grows beyond this
+     * size, Firestore SDK will start removing data that hasn't been recently used. The size is not
+     * a guarantee that the cache will stay below that size, only that if the cache exceeds the
+     * given size, cleanup will be attempted.
+     *
+     * <p>A default size of 100MB is used if unset. The minimum value to set is 1 MB.
+     *
+     * @return this {@code Builder} instance.
+     */
+    @NonNull
+    public Builder setSizeBytes(long size) {
       sizeBytes = size;
+      return this;
     }
   }
 
   private MemoryLruGcSettings(long size) {
     sizeBytes = size;
   }
 
+  /** Returns a new instance of {@link MemoryLruGcSettings.Builder} with default configurations. */
   @NonNull
   public static MemoryLruGcSettings.Builder newBuilder() {
     return new Builder();
   }
 
+  /**
+   * Returns cache size threshold for the memory cache. If the cache grows beyond this size,
+   * Firestore SDK will start removing data that hasn't been recently used. The size is not a
+   * guarantee that the cache will stay below that size, only that if the cache exceeds the given
+   * size, cleanup will be attempted.
+   *
+   * <p>By default, persistent cache is enabled with a cache size of 100 MB. The minimum value is 1
+   * MB.
```

Comment:
nit: "1 MB" -> "N,NNN,NNN" here as well (you can omit the units since it is documented to be in bytes)

Reply:
Done.